### ISSUE TITLE
Sections not showing on the home page if you only have one (same for tags)

### ISSUE THREAD

**Issue by Cyxo** [rafed/ramium#16]
If you only have one section (same for tags), the Sections part on the home page doesn't show up, due to this line in the template : https://github.com/rafed/ramium/blob/998f9689a55913acc51653c27e2c685abf06df83/layouts/partials/index/section-bullet.html#L4

I was kind of disappointed not seeing the Sections part although I only had one section (other than Posts). I think this should at least be a setting in the config.toml file to allow showing a single section on the home page.

**Comment by jiapei100** [comment]
@Cyxo Did you solve this problem?

**Comment by Cyxo** [comment]
Yeah, I managed to do so by replacing the `gt` with a `ge` in the line quoted above
